include/deal.II/base/mpi.h

```diff
@@ -362,7 +362,23 @@ namespace Utilities
     {
     public:
       /**
-       * Initialize MPI (and, if deal.II was configured to use it, PETSc) and
+       * Initialize MPI and, if deal.II was configured to use it, PETSc/SLEPc.
+       *
+       * @param[in,out] argc A reference to the 'argc' argument passed to
+       * main. This argument is used to initialize MPI (and, possibly, PETSc)
+       * as they read arguments from the command line.
+       * @param[in,out] argv A reference to the 'argv' argument passed to
+       * main.
+       *
+       * @note: It is recommended to use the constructor with three arguments
+       * to also set up the number of threads to use (see below), unless you
+       * need fine-grained control and want to call
+       * MultithreadInfo::set_thread_limit() yourself.
```

Comment:
This comment crucially omits what the function does with regard to threads. Nothing?